package.json

```diff
@@ -130,7 +130,7 @@
       }
     },
     "test-ci": {
-      "command": "npm run version-check && npm run flow:check && bin/config-check.js && better-npm-run jest --maxWorkers=2 --color=false --coverage",
+      "command": "npm run version-check && npm run flow:check && bin/config-check.js && better-npm-run jest --maxWorkers=2 --color=false --coverage --verbose",
       "env": {
```

Comment:
Can we remove that flag now?

Reply:
The ‘—verbose’ flag? No, we cannot. It’s what is allowing the tests to pass on node 16. See my comment above.

Reply:
That PR seemed to have random red and green builds so I was wondering. It would be good to understand why `--verbose` does change the outcome of the test suite, though. Although harmless, I don't think this is a good solution if we cannot explain it.

Reply:
To further clarify, the "It's working again" comment at https://github.com/mozilla/addons-frontend/pull/11037#issuecomment-989295100 was only in reference to the fact that the code I wrote broke SSR, which I commented on at https://github.com/mozilla/addons-frontend/pull/11037#issuecomment-989280606. It was not about the node 16 issue, which thus far I have only been able to fix by doing one of 3 things, either:
- add the `-i` flag
- add the `--verbose` flag
- remove the `--coverage` flag

Reply:
> That PR seemed to have random red and green builds so I was wondering. It would be good to understand why `--verbose` does change the outcome of the test suite, though. Although harmless, I don't think this is a good solution if we cannot explain it.

I agree, and mentioned this above, but I'm not sure what I can do at this point. There don't seem to be any other debugging options I can add. One thing I haven't tried, and maybe it will just magically fix the problem, is to update this patch to upgrade to jsdom 19 instead of 18. I'll give that a try and cross my fingers really hard.